aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs

```diff
@@ -328,9 +341,8 @@ proptest! {
 proptest! {
     #![proptest_config(ProptestConfig {
         // Cases are expensive, few cases is enough for these
-        // when trying to stress-test, increase (to 200 or more), and disable result cache.
-        cases: 10,
-        result_cache: prop::test_runner::basic_result_cache,
+        cases: if STRESSTEST_MODE { 1000 } else { 10 },
```

Comment:
nit: maybe we can have a unique place where we create proptest config for stresstest or not

Reply:
not sure what you have in mind

Reply:
we initialise proptest config in multiple places, thought this can be factored out

Reply:
yeah, I had a similar thought that maybe STRESSTEST_MODE can instead be an optional config, that contains a value (e.g. 1000 for cases, prop::test_runner::noop_result_cache for the cache_kind, etc)